crashlytics/CrashlyticsExample.xcodeproj/project.pbxproj

```diff
@@ -9,10 +9,6 @@
 /* Begin PBXBuildFile section */
 		1C9177A529BF382E0B10F21A /* GoogleService-Info.plist in Sources */ = {isa = PBXBuildFile; fileRef = 09DB1C43785994E8D625721E /* GoogleService-Info.plist */; };
 		86AC3FDA0C3B45EB493D8912 /* GoogleService-Info.plist in Sources */ = {isa = PBXBuildFile; fileRef = 09DB1C43785994E8D625721E /* GoogleService-Info.plist */; };
```

Comment:
Not part of these changes, but looks like you may have multiple references to the GoogleService-Info.plist in the project.

Reply:
On the Xcode files I only see one GoogleService-Info.plist file, so I manually removed the extra line of GoogleService-Info.plist from project.pbxproj. Since it still compiled successfully, is it a valid approach?

Reply:
Should be.